pkgs/games/grapejuice/default.nix

```diff
@@ -86,6 +86,16 @@ python3Packages.buildPythonApplication rec  {
 
     substituteInPlace src/grapejuice_common/models/settings_model.py \
       --replace 'default_wine_home: Optional[str] = ""' 'default_wine_home: Optional[str] = "${wine}"'
+
+    # Removed in 7.10.3. Required to set up the binary files.
+   substituteInPlace setup.py \
+      --replace 'install_requires=install_requires(),' 'install_requires=install_requires(),
+        entry_points={
+            "console_scripts": [
+                "grapejuice=grapejuice.cli.main:easy_install_main",
+                "grapejuice-gui=grapejuice.cli.gui:easy_install_main"
+            ]
+        },'
```

Comment:
AUR is doing `python3 -m grapejuice_packaging linux_package`
https://gitlab.com/brinkervii/grapejuice/-/blob/master/src/grapejuice_packaging/builders/linux_package_builder.py

Let's add a link to the file above the version to tell the updater to check that file for new `@build.task`'s which aren't in postInstall yet.

Reply:
Above which version? Line 90 (specifies a version a section was removed), line 22 (package version), or somewhere else? Any additional wording besides putting a link?

Reply:
No need now,  I moved to use the installation script

Reply:
Thanks. I didn't move to it because I couldn't figure out how to get it to work.

Sadly, there is a new problem though which I didn't run into yesterday. Opening Roblox shows "Roblox does not support Admin installation, please rerun RobloxPlayerInstaller without admin privilege". [It happens on other distributions as well](https://gitlab.com/brinkervii/grapejuice/-/issues/494), and may be a problem with Roblox itself. Regardless though, I can confirm those changes for Grapejuice work on a near-clean NixOS install.

Reply:
Looking at the conversations again, the only *current* way to resolve the issue seems to be using WINE Staging 8.13, or a custom-compiled version of wine-tkg that doesn't exist yet. It has the advantage of including a patch to get Vulkan working with Roblox Studio at the cost of... well... not existing, yet.